### DISCUSSION TITLE
3 Interesting Ideas/Suggestions

### DISCUSSION THREAD

**Discussion by M0REKZ** [WohlSoft/PGE-Project#432] [Ideas]
Well, my fingers hurt...

a suggestion:

1. Suggestion: Make icon cats different breed.
All my life i thinked the gray cat are the same everywhere until i see https://wohlsoft.ru/pgewiki/Gyro , you can make different colors and designs for each cat, for example: the builder cat could be striped and orange, the cat with glasses could be white with black spots, etc.

And some ideas that occurred to me because I don't have a life ... well yes I have but they occurred to me because yes:

2. Idea: enable users to add their own level formats with an importer and exporter script
I had already mentioned a similar idea, but allowing people to make their own level translator would be very interesting.
An example: you can enable users to write a lua, python, idklanguagexd, that can translate the level data to PGE Data, when PGE need to open a Weird Level format, can run the Importer program, and when the user need to save this, PGE will run the corresponding exporter

3. Idea: Make an interface for modifing/creating Config Packs
It could just be more comfortable for anyone who wants to make a config pack

They are just ideas that occurred to me, discard them if you want, but preferably do number 1, you will save thousands of lives avoiding confusion Xd

**Comment by Wohlstand** [comment]
1. I'll ask the Tails608 who recently had to draw new icons, looks good idea.
2. Looks like giving an ability to support file formats out of PGE-FL's box. That would work with Editor only, and will work as "open from alien format" or "export as alien format", because they aren't mainstream formats are part of PGE-FL. But yeah, together with parsing of formats by theme selves, users will probably need to map item sets to make them being properly shown with a target config pack.
3. This idea I keep for a while, and it's absolutely good idea! :fox_face: I think to build this later.

**Comment by M0REKZ** [comment]
Nice!

**Comment by Wohlstand** [comment]
Icon examples with different breed, made by **Tails608** a hour ago:
![изображение](https://user-images.githubusercontent.com/6751442/88223195-bcd91180-cc6f-11ea-8db3-dc81f2c6af56.png)

**Comment by M0REKZ** [comment]
pretty cats, so no one will think it are the same cat :)
